artifacts/manifests/podspec.yaml

```diff
@@ -14,7 +14,7 @@ initContainers:
       memory: 200Mi
     limits:
       cpu: 2
-      memory: 1.6Gi
+      memory: 1639Mi
```

Comment:
Can you also add a comment stating why this value is added. Users going through the spec should not get confused with a random number popping up for the memory limit.